src/ui_interface.rs

```diff
@@ -243,7 +243,11 @@ pub fn set_peer_option(id: String, name: String, value: String) {
 
 #[inline]
 pub fn using_public_server() -> bool {
-    crate::get_custom_rendezvous_server(get_option_("custom-rendezvous-server")).is_empty()
+    if hbb_common::config::RS_PUB_KEY == "OeVuKk5nlHiXp+APNn0Y3pC1Iwpwn44JGqrQCsWqmBw=" {
```

Comment:
Put "OeVuKk5nlHiXp+APNn0Y3pC1Iwpwn44JGqrQCsWqmBw=" here again is not good, we need a clever way (macro again) to avoid duplication.

Reply:
so I did it via checking at compile time for the existence of the env variable as well as checking for the custom server as before, that should resolve this concern